.github/pull_request_template.md

```diff
@@ -11,9 +11,9 @@ Describe how this change was tested.
 - [ ] **Update the title** of the PR to be succinct and less than 50 characters
 - [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
 - [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
-- [ ] Tag the PR with Azure API for FHIR if this will release to the Azure API for FHIR managed service (CosmosDB or common code related to service)
-- [ ] Tag the PR with Azure Healthcare APIs if this will release to the Azure Healthcare APIs managed service (Sql server or common code related to service)
-- [ ] CI is green before merge
+- [ ] Tag the PR with **Azure API for FHIR** if this will release to the Azure API for FHIR managed service (CosmosDB or common code related to service)
+- [ ] Tag the PR with **Azure Healthcare APIs** if this will release to the Azure Healthcare APIs managed service (Sql server or common code related to service)
+- [ ] CI is green before merge [![Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_apis/build/status/CI%20Build%20%26%20Deploy?branchName=main)](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
```

Comment:
Thoughts on spacing this to the next line?
![image](https://user-images.githubusercontent.com/197221/194678440-51bf23c0-468a-44a2-9d68-49155bc87e0c.png)


```suggestion
- [ ] CI is green before merge 
- [![Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_apis/build/status/CI%20Build%20%26%20Deploy?branchName=main)](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main)
```

Reply:
I thought having it on the same line helped link the two together. Since the squash merge requirements were on their own line I thought this looked better. I'm open to other thoughts.

Reply:
Its only a nit, I'm also ok starting with what you have in the PR